ansible/roles/openshift_udp_limits/tasks/main.yml

```diff
@@ -75,7 +65,9 @@
       after: "id_outbound_udp_filtering"
       regex: "id_outbound_udp_rule_1_"
       block: |-
-        -A oo-udp -d {{ servicenetwork }} -o eth0 -p udp -m udp --dport 53 -j RETURN
+        {# Note: sdodson says that pod DNS resolution happens via dnsmasq running on the node. That's
+                 why we only have DNS traffic to the node whitelisted prior to rate limiting. #}
+        -A oo-udp -d {{ oo_private_ip }}/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
```

Comment:
@twiest likewise, is it a mistake to use `oo_private_ip` here in the role, and should we instead pass it in from the tech profile?